### ISSUE TITLE
Led number issue, controller sub type request

### ISSUE THREAD

**Issue by djlastnight** [shauleiz/vXboxInterface#1]
Hi, I just created account to say Thank You for creating such an Amazing API!
I use it at my application with almost no problems. The GetLedNumber() always returns 1.
I'm using P/Invoke to call your code from C#. Here is the function signature:
`        [DllImport("vXboxInterface.dll", EntryPoint = "GetLedNumber", CallingConvention = CallingConvention.Cdecl)]
        private static extern bool GetLedNumber(uint userIndex, out byte ledNumber);`

I can live without the led number, but I want to ask you for a feature to implement.
Can you please provide a function, which sets the virtual controller sub type (subtypes listed here: https://msdn.microsoft.com/en-us/library/windows/desktop/hh405050%28v=vs.85%29.aspx)?

Thank you again!
Best Regards,
djlastnight

**Comment by shauleiz** [comment]
Thank you,
This interface is only temporary, or proof of concept.
Once we finalize the driver I will come back to it.

I know that on Windows 7 the Led number is meaningless. Is this the case?

As for your suggestion:
Interesting. Need to see if it is feasible. Probably a change in the driver.

**Comment by shauleiz** [comment]
Thank you,
This interface is only temporary, or proof of concept.
Once we finalize the driver I will come back to it.

I know that on Windows 7 the Led number is meaningless. Is this the case?

As for your suggestion:
Interesting. Need to see if it is feasible. Probably a change in the driver.

**Comment by djlastnight** [comment]
Yes, I tested it under Windows 7 (x86 and x64) along with real xbox 360 controller.

Thank you for the fast answer.
I hope you will add my request to your ToDo list :)

Best wishes,
djlastnight
